src/Razor/src/Microsoft.VisualStudio.Editor.Razor/Documents/EditorDocumentManagerListener.cs

```diff
@@ -153,20 +155,21 @@ internal void ProjectManager_Changed(object sender, ProjectChangeEventArgs e)
         }
 
         private void Document_ChangedOnDisk(object sender, EventArgs e)
+        {
+            _ = Document_ChangedOnDiskAsync((EditorDocument)sender, CancellationToken.None);
```

Comment:
💡 This is an untracked fire-and-forget operation, which means tests will not be able to detect it or wait for it to complete.